common/time/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+#![forbid(unsafe_code)]
+
+pub mod time {
+    use std::time::{Duration, SystemTime};
+
+    // Gives the duration since the Unix epoch, notice the expect.
+    pub fn duration_since_epoch() -> Duration {
+        SystemTime::now()
+            .duration_since(SystemTime::UNIX_EPOCH)
+            .expect("System time is before the UNIX_EPOCH")
+    }
+
+    // This is just an alias to make code more readable
+    pub fn current_time() -> Duration {
```

Comment:
I'd suggest removing this function, this seems unnecessary

Reply:
It's a synonym with a name that is better depending on the context. 
I can remove it, but it might be strange sometimes to call `duration_since_epoch` when you want the current time. 
I thought it would help with readability. No?

Reply:
yeah I see what you're saying, I feel like we in general tend to avoid aliases, but if you think it's really better with the alias then I'll let you decide. Another way could be to rename `duration_since_epoch` also? I never found it clear, imo `get_current_unix_time` or something like that would be clearer in all cases.